Automattic/wp-calypso
# WP.com block editor: Deprecate CoBlocks buttons block

#### Changes proposed in this Pull Request

Overrides the CoBlocks Buttons block settings to make it non-insertable (so it won't collide with Core's new Buttons block).

Further reading: paYKcK-rn-p2

#### Testing instructions

* Apply D37423-code, sandbox `widgets.wp.com` and a simple site.
* Load the block editor.
* Try finding the CoBlocks 'Buttons' block in the block inserter: It shouldn't be there.
* Previously created posts that include the CoBlocks 'Buttons' block should continue to work.
